.github/instructions/careui.instructions.md

```
@@ -55,19 +55,17 @@ import { Card } from "@/components/ui/card";
 ## Healthcare Design Requirements
 
 ### Accessibility
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)